client/src/components/RequestAccess/RequestAccessForm.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Formik, Form } from 'formik';
+import { Alert, Button, FormGroup, Spinner } from 'reactstrap';
+import { FormField } from '_common';
+import * as Yup from 'yup';
+import './RequestAccessForm.module.scss';
+
+const formSchema = Yup.object().shape({
+  username: Yup.string().required('Required'),
+  password: Yup.string().required('Required'),
+  problem_description: Yup.string().required('Required')
```

Comment:
The button is valid and clickable on load without entering any data. Perhaps we need to set the defaultValues to `null`
<img width="1022" alt="Screen Shot 2021-12-07 at 1 54 34 PM" src="https://user-images.githubusercontent.com/20326896/145097090-d27b1adb-d8e6-4bf5-aed4-1c17fc5a8c3d.png">

Reply:
Using validateOnMount property instead.